### PR TITLE
Feature/cica 929/add readme

### PR DESCRIPTION
**Pull request by armoj**
First draft of cw readme.

### CODE REVIEW

**Comment by neil-stephen-mcgonigle** [comment]
Do we need 5 WIP commits? 😂

**Comment by armoj** [comment]
> Do we need 5 WIP commits? 😂

Forgot to push the rebase

File: README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+Make sure that this repo is installed in a sibling directory to the Data Capture Service API. This directory structure is already configured in the `docker-compose.yml` file that are used to run the containers.
+
+Clone the Cica-Web repo, and `npm install`
+
+## Using Cica-Web
+In order to fully run and use the web app, you will need to clone and install the [Data Capture Service API](https://github.com/CriminalInjuriesCompensationAuthority/data-capture-service). The neccessary `docker-compose.yml` file is found in the dev-utilities repo.
+
+This Docker Compose configuration will assume that there is a cica-web, and data-capture-service directory, both with the respective installed repos in them for it to run correctly.
+
+> Full instructions on the required directory structure and configuration is found in the `Docker-compose-setup-for-CW,-DCS,-Postgres` Utilities Wiki article.
```
Comment (neil-stephen-mcgonigle) [inline README.md:31]: Should we mention that this is a private repo/wiki?

Comment (armoj) [reply README.md:31]: Not sure. Given that this information is for internal contributors only, they would know of this wiki anyway. So left it out.

Comment (neil-stephen-mcgonigle) [reply README.md:31]: yeah, I am not sure either. The README will be viewable by anyone though.

Comment (armoj) [reply README.md:31]: yeah so I was trying to not expose names/structures of repos 🤝

Comment (neil-stephen-mcgonigle) [reply README.md:31]: yeah, all good I was not suggesting to expose the names just make it clear that this information is not available. Saves someone clicking/looking for it.

Comment (armoj) [reply README.md]: Added a note to the relevant line 🤝

**Review by neil-stephen-mcgonigle — APPROVED** [review]

**Pull request merged**